### ISSUE TITLE
在单张24GB显存显卡上训练，无法收敛

### ISSUE THREAD

**Issue by deltawen** [dasongli1/Shift-Net#16]
Using an RTX3090 Ti 24GB, train our-s, n_sequence=12 on the DAVIS dataset, but looking at the loss, we find that the Loss has been fluctuating wildly.
![image](https://github.com/dasongli1/Shift-Net/assets/83681909/0fb72126-9a9b-4b5f-8898-68dcee314843)

**Comment by 6liz6** [comment]
> Using an RTX3090 Ti 24GB, train our-s, n_sequence=12 on the DAVIS dataset, but looking at the loss, we find that the Loss has been fluctuating wildly. ![image](https://private-user-images.githubusercontent.com/83681909/332000996-0fb72126-9a9b-4b5f-8898-68dcee314843.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc5MDU2ODAsIm5iZiI6MTcxNzkwNTM4MCwicGF0aCI6Ii84MzY4MTkwOS8zMzIwMDA5OTYtMGZiNzIxMjYtOWE5Yi00YjVmLTg4OTgtNjhkY2VlMzE0ODQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA5VDAzNTYyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1YzU4ODY5ZGNhYWNlZTQyYzg4MTE1MGU4M2ZlZGQ1MDgyYmM5NTZmYmRlOWU4OTI0N2RhNGY2NDI1YTQ1MmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RijH1yrO-U7T_gAiAJTKNdVSDNjesX9OSxmdfX4UHlw)

I'm in the same boat, I can't seem to converge when I train on the DVD dataset, have you solved this problem yet? I've tried adjusting the learning rate several times and it doesn't seem to work.
